lib/modules/filteReddit.js

```diff
@@ -746,6 +747,18 @@ module.go = function() {
 			toggleNsfwFilter(toggle, true);
 		}
 	);
+
+	CommandLine.registerCommand(
+		(cmd, val) => {
+			if (!(/^f(?:ilter)?/).test(cmd)) return false;
+			return val ||
+				SelectedEntry.selectedThing() && SelectedEntry.selectedThing().getSubreddit() ||
+				currentSubreddit();
```

Comment:
How about adding a ` || ''` fallback, for if selected entry is off or on a page that doesn't have anything to select or a subreddit context?